Svenbjorn/whatson
# Task/api integrations

## Setup

You will need to setup api keys in the following file: `views/whatson/vault/apiKeys.json`

## How to test

The following features have gotten new api keys (.env file must be updated!)

    mapbox
    google maps (For street view purpose)

Test these features to ensure they work.
